### DISCUSSION TITLE
Factions and how they came into being

### DISCUSSION THREAD

**Discussion by NetSysFire** [CleverRaven/Cataclysm-DDA#60898] [Ideas]
All the giant spider, wasp and dermatik nests just popped into being. There is a newspaper entry from a year ago (in-game time) talking about bees 2-3 times their normal size but not aggressive (yet). But there appear to be no other reports of all the other factions which span from these rat caves to the insects, to slimes.

There needs to be some lore about when they actually became as common as they are now and their interactions when society did not collapse yet.

This is part of the Living World project (https://github.com/orgs/CleverRaven/projects/5).

**Comment by worm-girl** [comment]
A lot of these non-zombifying animals are scavengers or predators. In my mind, they're getting a lot of XE037 in their systems because of their diet, which builds up similar to mercury in oceanic predators. As more zombified/mutated animals and people come into being, they spread concentrated XE037 to scavenger insects, which pass it along to predators. In this way, we can tie the spread of giant insects to the proliferation of human zombies. A year ago, the wasps were certainly getting bigger, but they really blew up in the last couple of months, when things were too far gone for humanity to really notice and react to them.

Additionally, because XE037 is semi-sentient and has goals, it could be "purposefully" taking advantage of lower lifeforms to make life difficult for large animals. There's lore stating it can't zombify very small animals, but if it can mutate them into extremely dangerous forms, they still serve its purposes by killing more large animals and thereby either generating zombies or weakening resistance to full global takeover. Wasps don't get along with zombies, but if they take out the humans, that's a net positive for the Blob, which can destroy the wasps later at its leisure if it cares to. This would be a better explanation for herbivorous insects like bees, aphids, and grasshoppers - these animals are necessary for human agriculture, and mutating them would greatly destabilize human societies.

This timeline would mean there wasn't really time for the mainstream media to write a lot of stories about dog-sized wasps taking over towns, because by then everyone was so feral or distracted by the zombie outbreak that very little reliable news was getting printed. The game begins in spring, probably shortly after many of the first generation of enemy arthropods have been born, and pre-cataclysm society may not have gotten a good look at them. It's a brave new world!

**Comment by LyleSY** [comment]
Mechanically are we talking about lab reports? Snippets and interesting objects at nests? NPC survivor stories? Something else I can't think of right now?
